src/rosdep2/lookup.py

```diff
@@ -328,7 +328,7 @@ def get_resources_that_need(self, rosdep_name):
     @staticmethod
     def create_from_rospkg(rospack=None, rosstack=None,
                            sources_loader=None,
-                           verbose=False):
+                           verbose=False, dependency_types=[]):
```

Comment:
I wouldn't recommend this because of mutable default arguments:

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Even if you only read from it.

Reply:
Wacky gotcha! Nice call-out